TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
```

Comment:
what is the purpose of that class?

Reply:
To make a multipurpose method ExecuteQuery which processes any query to the database.